chore: asset folder for icons

Aadarsh-shrivastava opened the pull request:
Add assets folders consisting of png, jpg and SVGs for icons and logos

Aadarsh-shrivastava committed:
add: asset folder for icons

shah-harshit commented:
Update the PR title to `chore: asset folder for icons`

shah-harshit reviewed (commented):
Why do we have 2 logos for some assets? please remove the assets if not required.
1. logo.jpeg and logo.png
2. table-alt-svgrepo-com.svg

Aadarsh-shrivastava committed:
chore : remove duplicate files

Aadarsh-shrivastava committed:
chore : rename all the asset files

Aadarsh-shrivastava commented:
> Why do we have 2 logos for some assets? please remove the assets if not required.
> 
> 1. logo.jpeg and logo.png
> 2. table-alt-svgrepo-com.svg

I have removed all the duplicate files and renamed assets
